stdlib/3/tkinter/__init__.pyi

```diff
@@ -252,9 +252,13 @@ class Misc:
     def tk_focusFollowsMouse(self): ...
     def tk_focusNext(self): ...
     def tk_focusPrev(self): ...
-    def after(self, ms, func: Optional[Any] = ..., *args): ...
-    def after_idle(self, func, *args): ...
-    def after_cancel(self, id): ...
+    @overload
+    def after(self, ms: int, func: None = ...) -> None: ...
+    @overload
+    def after(self, ms: Union[int, Literal["idle"]], func: Callable[..., None], *args: Any) -> str: ...
```

Comment:
Does `func` really have to return `None`? In case the return value is ignored, use `Any`.

Reply:
I've didn't know that ignored return types in callback functions should be `Any`. I have changed the return types in this PR and I think I'll make another PR that fixes callbacks I have created earlier.